app/services/csv_school_moves.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+class CSVSchoolMoves
+  include ServicePattern
+
+  HEADERS = %i[
+    NHS_REF
+    SURNAME
+    FORENAME
+    GENDER
+    DOB
+    ADDRESS1
+    ADDRESS2
+    ADDRESS3
+    TOWN
+    POSTCODE
+    COUNTY
+    ETHNIC_OR
+    ETHNIC_DESCRIPTION
+    NATIONAL_URN_NO
+    BASE_NAME
+    STARTDATE
+    STUD_ID
+    DES_NUMBER
+  ].freeze
+
+  def initialize(school_move_log_entries)
+    @school_move_log_entries = school_move_log_entries
+  end
+
+  def call
+    CSV.generate(headers: HEADERS, write_headers: true) do |csv|
+      school_move_log_entries.each do |log_entry|
+        csv << row(log_entry.school, log_entry.patient)
+      end
+    end
+  end
+
+  private
+
+  attr_reader :school_move_log_entries
+
+  def row(school, patient)
+    [
+      patient.nhs_number,
+      patient.family_name,
+      patient.given_name,
+      patient.gender_code.humanize,
+      patient.date_of_birth.to_fs(:govuk),
+      patient.address_line_1,
+      patient.address_line_2,
+      nil,
+      patient.address_town,
+      patient.address_postcode,
+      nil,
+      nil,
+      nil,
+      school_urn(school, patient),
+      school&.name,
+      school&.created_at&.to_fs(:govuk),
+      nil,
+      nil
+    ]
+  end
+
+  def school_urn(school, patient)
```

Comment:
I think we should use the existing method in `Reports::ExportFormatters` to do this: https://github.com/nhsuk/manage-vaccinations-in-schools/blob/688c6bab2308b87ff3ff5c35b7f7cc3e4aacfbfa/app/lib/reports/export_formatters.rb#L6-L14